app/code/Magento/Inventory/Setup/InstallSchema.php

```diff
@@ -0,0 +1,228 @@
+
+
+    /**
+     * Option keys for column options
+     */
+    const OPTION_IDENTITY = 'identity';
+    const OPTION_UNSIGNED = 'unsigned';
+    const OPTION_NULLABLE = 'nullable';
+    const OPTION_PRIMARY = 'primary';
+    const OPTION_DEFAULT = 'default';
+    const OPTION_TYPE = 'type';
+    const OPTION_LENGTH = 'length';
+
+    /**
+     * @param SchemaSetupInterface $setup
+     * @param ModuleContextInterface $context
+     */
+    public function install(SchemaSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $installer = $setup;
+        $installer->startSetup();
+
+        $tableNameSourceEntity = $installer->getTable(InstallSchema::TABLE_NAME_SOURCE);
+        if (!$installer->getConnection()->isTableExists($tableNameSourceEntity)) {
+
+            $table = $installer->getConnection()->newTable($tableNameSourceEntity);
+
+            $options = [
+                InstallSchema::OPTION_IDENTITY => true,
+                InstallSchema::OPTION_UNSIGNED => true,
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_PRIMARY => true
+            ];
+            $table->addColumn(SourceInterface::SOURCE_ID, Table::TYPE_INTEGER, null, $options, 'Source ID');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::NAME, Table::TYPE_TEXT, 255, $options, 'Source Name');
+
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::CONTACT_NAME, Table::TYPE_TEXT, 255, $options, 'Contact Name');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::EMAIL, Table::TYPE_TEXT, 255, $options, 'Email');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_UNSIGNED => true,
+                InstallSchema::OPTION_DEFAULT => 1
+            ];
+            $table->addColumn(SourceInterface::IS_ACTIVE,
+                Table::TYPE_SMALLINT,
+                null,
+                $options,
+                'Defines Is Source Active');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::DESCRIPTION, Table::TYPE_TEXT, 255, $options, 'Description');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::LATITUDE, Table::TYPE_TEXT, 255, $options, 'Latitude');
```

Comment:
changed it to decimal an used 8 decimals, used this https://en.wikipedia.org/wiki/Decimal_degrees as resource. I think that should fit all thinkable usecases.